numpy/ma/core.pyi

```diff
@@ -408,23 +408,23 @@ class MaskedArray(ndarray[_ShapeT_co, _DTypeT_co]):
     def shape(self) -> _ShapeT_co: ...
     @shape.setter
     def shape(self: MaskedArray[_ShapeT, Any], shape: _ShapeT, /) -> None: ...
-    def __setmask__(self, mask, copy=...): ...
+    def __setmask__(self, mask: _ArrayLikeBool_co, copy: bool = False) -> None: ...
     @property
-    def mask(self): ...
+    def mask(self) -> NDArray[bool_] | MaskType: ...
```

Comment:
I forgot what `MaskType` was just now, so had to look it up. Perhaps something like this could prevent that from happening again (I'm a goldfish).

```suggestion
    def mask(self) -> NDArray[MaskType] | MaskType: ...
```